src/store/reducer/index.ts

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+
+const createReducer = (asyncReducers: object) =>
+    combineReducers({
```

Comment:
since it's just `asyncReducers` only, maybe you can just do `combineReducers(asyncReducers)` instead. In fact, at that point, `createReducer` is just an alias for `combineReducers` and is maybe not needed.